packages/npm-script-allowlist/bin/hmpps-allow-scripts

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+import fs from 'fs'
+import npmRunScript from '@npmcli/run-script'
+
+const CONFIG_LOCATION = `${process.cwd()}/.allowed-scripts.mjs`
+const packageLockJson = JSON.parse(fs.readFileSync('package-lock.json', 'utf-8'))
+
+console.log(`Gathering configuration information from ${CONFIG_LOCATION}\n`)
+
+const configuredAllowlistModule = await import(CONFIG_LOCATION)
+const config = Object.assign({}, configuredAllowlistModule).default
+
+const configuredAllowlist = Object.entries(config.allowlist)
+const dependencyScriptsToRun = config.dependencyScriptsToRun || ['preinstall', 'install', 'postinstall']
+const localScriptsToRun = config.localScriptsToRun || ['preinstall', 'install', 'postinstall']
```

Comment:
slightly different in that you can override the script list to execute for both local and dependency scripts. (This could potentially be per dependency version but felt like overkill when we don't have a use case)

Reply:
Only vague suggestion was the defaults could be in their own `const` maybe?

Reply:
thanks, I'll tidy that up in a follow up